docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-related-resources/index.scss

```diff
@@ -1,8 +1,14 @@
+@forward 'usa-accordion';
 @forward 'usa-link';
 @use 'uswds-core' as *;
 @import '~Styles/sprites/svg-sprite';
 
 .cgdp-related-resources {
+	&__heading {
+		@include at-media('tablet') {
+			@include typeset-h2;
+		}
+	}
```

Comment:
Can we just make this `@include typeset-h2;` so that when  we turn off the accordion we don't miss this size as well?

This will keep it the same styling as `nci-heading-h2`.

Reply:
I think the solution to this was just to undo the heading class change all together.